app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -119,7 +137,7 @@ private String getSvmVector(TensorFlow tensorFlow, StudentImage studentImage){
      * Initialize TensorFlow model
      * @return
      */
-    private TensorFlow getInitializedTensorFlow(){
+    private synchronized TensorFlow getInitializedTensorFlow(){
         File modelFile = new File(AiHelper.getModelDirectory(), "vgg_faces.pb");
         if (!modelFile.exists()){
             String logMessage = "Model file: " + modelFile.getAbsolutePath() + " doesn't exist. Please copy it manually";
```

Comment:
TODO / prerequisite when preparing the tablet / Shall I provide a link to Google Drive or where?